include/blade/memory/devices/cpu/vector.hh

```diff
@@ -15,6 +15,8 @@ class BLADE_API Vector<Device::CPU, T, Dims> : public VectorImpl<T, Dims> {
         BL_CHECK_THROW(this->resize(dims));
     }
 
+    explicit Vector(const Vector<Device::CPU, T, Dims>& other) : VectorImpl<T, Dims>(other.data(), other.dims()) {}
```

Comment:
This can be dangerous, is it really necessary?

Reply:
I agree, but I'd like to double check how necessary just before merging...